test/node/bson_corpus_tests.js

```diff
@@ -48,40 +49,97 @@ function normalize(cEJ) {
   return JSON.stringify(JSON.parse(cEJ));
 }
 
-// tests from the corpus that we need to skip, and explanations why
+const parseErrorForDecimal128 = scenario => {
+  const skipRegex = /dqbsr|Inexact/;
+  for (const parseError of scenario.parseErrors) {
+    it(parseError.description, function () {
+      if (skipRegex.test(parseError.description)) {
```

Comment:
can we add ticket to unskip?

Reply:
This would also be covered by: jira.mongodb.org/browse/NODE-3637

Reply:
any objections to including it in a code comment here for easier reference? (same on the other place)

Reply:
tyvm :)